src/main/pages/che-7/contributor-guide/proc_defining-custom-branding-values-for-che-theia.adoc

```diff
@@ -34,9 +34,9 @@ Create a `product.json` file with a new name of the product, logo, description,
 ----
 <1> `name`: tab title for the *Welcome* page and the *About* dialog.
 <2> `icon`: icon for the *Welcome* page tab title.
-<3> `logo`: product logo for dark and light themes on the *Welcome* page (maximum height 80 px) and in the *About* dialog (maximum height 100 px). Use an image with a transparent background. Define a relative path, an absolute path, or a URL to an image. 
+<3> `logo`: product logo for dark and light themes on the *Welcome* page (maximum height 80px) and in the *About* dialog (maximum height 100px). Use an image with a transparent background. Define a relative path, an absolute path, or a URL to an image.
```

Comment:
This was correct. There should be a space between numbers and units (see IBM SG, pg. 167).
```suggestion
<3> `logo`: product logo for dark and light themes on the *Welcome* page (maximum height 80{nbsp}px) and in the *About* dialog (maximum height 100{nbsp}px). Use an image with a transparent background. Define a relative path, an absolute path, or a URL to an image.
```

Reply:
https://www.ibm.com/developerworks/library/styleguidelines/index.html
> Use no space between the numeral and the abbreviation, and use the same abbreviation for both singular and plural unless 

*shrug*

Reply:
To avoid confusion, I'll just not use the abbreviation.

Reply:
That's very strange :) The regular IBM SG says:
![image](https://user-images.githubusercontent.com/6623103/77168171-a8b1b780-6ab7-11ea-8c6c-6e53cfd53f6f.png)
We'll need to revisit what to take as the definitive resource.

Reply:
We reference IBM styleguide circa 2011 while Developerworks is also no longer maintained, at least it's a little bit fresher - 2014 :D 

The third edition of the styleguide (2014) is this https://www.amazon.com/Developing-Quality-Technical-Information-Handbook/dp/0133118975/ but I personally never received it, I only have the 2011 copy. 
I'd say the paper copy should be the source of truth but only the most current one...